### PR TITLE
Update to Sylius 1.4

### PR DESCRIPTION
**Pull request by Zales0123**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "require": {
         "php": "^7.2",
         "knplabs/knp-snappy-bundle": "^1.5",
-        "sylius/sylius": "1.3.*",
+        "sylius/sylius": "~1.4.0",
```
Comment (bartoszpietrzak1994) [inline composer.json]: In other plugins we require `sylius/sylius` in `^1.4` version. Requiring it in `~1.4.0` version makes me wonder what approach is more suitable for us, WDYT?

Comment (Zales0123) [reply composer.json]: `^1.4` 👌

File: behat.yml.dist [hunk 2]
```diff
@@ -1,24 +1,47 @@
...
-                path: tests/Application/Kernel.php
-                bootstrap: ~
 
-        Lakion\Behat\MinkDebugExtension:
-            directory: etc/build
-            clean_start: false
-            screenshot: true
+        FriendsOfBehat\VariadicExtension: ~
+
+        FriendsOfBehat\SuiteSettingsExtension:
```
Comment (pamil) [inline behat.yml.dist]: This can be removed - they're the default settings.

**Comment by bartoszpietrzak1994** [comment]
Thank you, Mateusz! :1st_place_medal:

**Pull request merged**